defi/package.json

```diff
@@ -63,15 +63,15 @@
     "@defillama/adapters": "file:./DefiLlama-Adapters",
     "@defillama/dimension-adapters": "file:./dimension-adapters",
     "@defillama/emissions-adapters": "file:./emissions-adapters",
-    "@defillama/sdk": "^3.0.1",
+    "@defillama/sdk": "^3.0.51",
     "@supercharge/promise-pool": "^2.3.2",
     "axios": "^0.27.2",
     "bignumber.js": "^9.0.1",
     "dotenv": "^16.0.1",
     "ethers": "^5.7.2",
     "mysql2": "^2.3.3",
     "node-fetch": "^2.6.1",
-    "pg": "^8.8.0",
```

Comment:
did you confirm that all `pg` usage was moved? there might be some usage in sub modules

Reply:
Yeah I cant see any dependencies